UX-STANDARDS.md

```diff
@@ -0,0 +1,11 @@
+# UX Standards
+
+These standards include some basic user experience guidelines we align to on the marketing website, docs, and the registry.
+
+- Every page should have exactly one primary action.
+- Use descriptive text for button labels and links, and avoid labels such as "Learn more".
```

Comment:
I actually think labels like these -- "Get started", "Learn more" -- are fine when the context is clear, especially on buttons (and indeed we do this ~everywhere), but I agree that in prose, we should definitely be specific. Maybe some examples would help make this distinction clearer?

Reply:
this goes against user experience best practices. can you link me to some sources that document that these best practices have changed?

Reply:
I agree with using descriptive text for button labels and links. Here's a little something I came across regarding this: https://www.accessguide.io/guide/descriptive-link-text
It gives examples of how we can compromise, for example, instead of saying "Learn more" we could say "Learn more about Pulumi" or whatever we want the user to learn more about. Possibly "Get started" could be the exception? Since I'm thinking that in situations where we use that, the context would be very clear and obvious.

Reply:
ya, get started should be an exception, especially because we have indexed our SEO so much on it

Reply:
I'm not sure I agree about every page having one primary action... I think this is a good rule to have in general, but couldn't there be instances/exceptions where we have more informational type pages where really the only action a user might take is to go back to the previous page?

Reply:
@jonelleb absolutely! let me fix this up to say "no more than" for the case you describe in which we would not have a primary button. though i would like us to strive to get to a place where we are building pages intentionally around a single primary action.